doc/cygnus-ngsi/flume_extensions_catalogue/last_data_function.md

```diff
@@ -181,7 +207,7 @@ Notice the value `4wheels2` on the column `fiwareServicePath` remained after the
 ### MYSQL
 
 `INSERT INTO sub_5dde9_last_data 
-(recvTime,fiwareServicePath,entityId,entityType,fillingLevel,fillingLevel_md) 
+(entityId,entityType,fiwareServicePath,recvTime,fillingLevel,fillingLevel_md) 
```

Comment:
We haven't tested anything on the MySQL sink. I think the aggregation will be also affect it (due to the needed changes in NGSIMySQLSinkTest.java to keep tests passing) but not fully sure.

Reply:
NTC (informative)